rotkehlchen/db/filtering.py

```diff
@@ -674,6 +675,12 @@ def make(
                 DBStringFilter(and_op=True, column='location_label', value=location_label),
             )
 
+        if location == Location.KRAKEN:
+            if from_ts is not None:
+                from_ts = Timestamp(from_ts * KRAKEN_TS_MULTIPLIER)
+            if to_ts is not None:
+                to_ts = Timestamp(to_ts * KRAKEN_TS_MULTIPLIER)
+
```

Comment:
We have a problem in the history events tables since the precision of the kraken events for the timestamp is higher. We either have to agree on using the same precision for all the events or update them. Otherwise queries by timestamp range in that table don't work properly

Reply:
yeah I know. I think going with the highest precision available (so kraken's is fine). As long as we deal with ints of the same precision there should be no difference.